### PR TITLE
#250-emptycontent-views

### PR DESCRIPTION
**Pull request by Nebri**
Signed-off-by: Ryan Fletcher <ryan.fletcher@codepassion.ca>

* Resolves: # 250
* Target version: master 

### Summary
Added empty content messages to board listings.

### TODO

- [] Add empty content messages to board listings.

### Checklist

- [ ] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by Nebri** [comment]
I'll make a quick revision as per [@jancborchardt's comment](https://github.com/nextcloud/deck/issues/250#issuecomment-414814356)

This will solve issue #250

File: js/controller/ListController.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ var ListController = function ($scope, $location, $filter, BoardService, $elemen
 	$scope.boardservice = BoardService;
 	$scope.updatingBoard = null;
 
-	var filterData = function () {
+	var filterData = function () {
```
Comment (juliushaertl) [inline js/controller/ListController.js]: We don't need additional whitespace :wink:

File: templates/part.boardlist.php [hunk 2]
```diff
@@ -23,7 +31,7 @@
 			<td></td>
 		</tr>
 		</thead>
-		<tbody>
+		<tbody>
```
Comment (juliushaertl) [inline templates/part.boardlist.php]: Whitespace.

File: css/style.scss [hunk 3]
```diff
@@ -34,6 +34,11 @@ $color-darkgrey: nc-darken($color-main-background, 32%);
 /**
  * General styles
  */
+
+.center-text {
```
Comment (juliushaertl) [inline css/style.scss]: This doesn't seem to be used anywhere.

**Comment by juliushaertl** [comment]
Merged :rocket:

**Pull request merged**